src/parser/jobs/whm/modules/Thinair.tsx

```diff
@@ -0,0 +1,143 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import Rotation from 'components/ui/Rotation'
+import {getDataBy} from 'data'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import React, {Fragment, ReactNode} from 'react'
+import {Accordion} from 'semantic-ui-react'
+
+type ThinAirRecord = {
```

Comment:
Please use an interface instead to be consistent with the project style.

```suggestion
interface ThinAirRecord {
```